openqabot/syncres.py

```diff
@@ -70,6 +70,10 @@ def filter_jobs(data) -> bool:
             )
             return False
 
+        if self.client.is_devel_group(data["group_id"]):
+            logger.info("Devel job %s in group %s" % (data["id"], data["group"]))
```

Comment:
Better make that debug

Reply:
this isn't bug , but one of many correct states

Reply:
Yes, but "info" should be used for something that is noteworthy and shouldn't appear that often. I assume this line would be logged quite often.

I would prefer if you could not call a conversation "resolved" in github when we haven't found a resolution yet.

Reply:
oh, wait. You are just moving the code. So, right, keep as is for now. We can still change this in another change but this is unrelated to a refactoring.